### PR TITLE
Select/Update/DateTime Bechmark Improvements

### PR DESCRIPTION
**Pull request by stanbrub**
- Rewrote select/update/view/update_view formula tests to be more obviously consistent.
- Removed former update tests and packages where appropriate
- Re-scaled DateTimeUtils tests to remove some variability between runs

Notes for Reviewer:
- Mainly looking for whether or not the benchmarks are capturing the difference between these operations when inline formulas are used.

### CODE REVIEW

File: src/it/java/io/deephaven/benchmark/tests/standard/sort/SelectDistinctTest.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /* Copyright (c) 2022-2023 Deephaven Data Labs and Patent Pending */
-package io.deephaven.benchmark.tests.standard.select;
+package io.deephaven.benchmark.tests.standard.sort;
```
Comment (rcaudy) [inline src/it/java/io/deephaven/benchmark/tests/standard/sort/SelectDistinctTest.java:2]: Why sort? Select distinct is actually an aggregation.

File: src/it/java/io/deephaven/benchmark/tests/standard/formula/InlineFormulaTest.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+import org.junit.jupiter.api.*;
+import io.deephaven.benchmark.tests.standard.StandardTestRunner;
+
+/**
+ * Standard tests for running basic formulas (not functions) inline for select, update, view, update_view, etc. These
+ * tests aggregate the resulting values to make sure the formulas are actually run (as in the case of views). Theses
+ * tests are meant to be compared, and so use the same data.
+ */
+public class InlineFormulaTest {
```
Comment (rcaudy) [inline src/it/java/io/deephaven/benchmark/tests/standard/formula/InlineFormulaTest.java:12]: Certainly seems cleaner and less duplicative, but haven't you dropped several cases in this transformation?

File: src/it/java/io/deephaven/benchmark/tests/standard/sort/SelectDistinctTest.java (hunk 1 above)
Comment (stanbrub) [reply src/it/java/io/deephaven/benchmark/tests/standard/sort/SelectDistinctTest.java:2]: Moved

File: src/it/java/io/deephaven/benchmark/tests/standard/formula/InlineFormulaTest.java (hunk 2 above)
Comment (stanbrub) [reply src/it/java/io/deephaven/benchmark/tests/standard/formula/InlineFormulaTest.java:12]: The purpose of those tests was to test the difference between doing the same inline formulas in update, update_view, select, and view.  Doing the _2 Calc 2 Col_ tests seemed sufficient.  The ones I removed were _1 Calc 2 Col_ tests and _int-vs-floats_.  Would those tests shed light on the difference between the operations?

**Review by rcaudy — APPROVED** [review]

**Pull request merged**